lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
```

Comment:
NIT: the ``view-incourse`` makes sense here for now, but I'd prefer if we can down the road split of "out of course" views (dashboard, account, course discovery, etc.) from in a course. Perhaps at that time, this class will be obselete.